### ISSUE TITLE
make test errors out on user-agent test in Singularity 3.6.0-rc.3

### ISSUE THREAD

**Issue by novosirj** [apptainer/singularity#5297]
### Version of Singularity:

What version of Singularity are you using? Run:

```
[novosirj@amarel-test1 singularity]$ /mnt/scratch/novosirj/opt/sw/packages/gcc-4_8/singularity/3.6.0-rc.3/bin/singularity version
3.6.0-rc.3
```

### Expected behavior

Didn't have problems with `make test` in previous versions; passed completely, with prefix either on or off GPFS. Now it fails both ways. I tried building and installing to XFS, just in case there was a problem with GPFS as a backing filesystem. My home directory remains on GPFS.

### Actual behavior

```
...
=== RUN   TestSingularityVersion
--- PASS: TestSingularityVersion (0.00s)
PASS
coverage: 88.9% of statements
ok      github.com/sylabs/singularity/pkg/util/user-agent       1.099s  coverage: 88.9% of statements
?       github.com/sylabs/singularity/scripts   [no test files]
FAIL
```

### Steps to reproduce this behavior

Nothing special. Tried to build with both "go" 1.13.10 and 1.13.11, configuration is nothing special, build uses the example script on the website that downloads "go" and Singularity, extracts them, and runs the build process. My specific build commands are like so:

```
./mconfig --prefix=/mnt/scratch/novosirj/opt/sw/packages/gcc-4_8/singularity/$VERSION \
        --localstatedir=/var \
        && cd builddir ; make -j32 && sudo -E make install && sudo ln -s /mnt/scratch/novosirj/opt/sw/packages/gcc-4_8/singularity/$VERSION/bin/singul
arity /usr/bin/singularity && make test && sudo rm -f /usr/bin/singularity
```

I only do that creation/deletion of the link because the install PATH is not in the system PATH. I think later versions allow you to pass a PATH and so I could make a change there.

### What OS/distro are you running

```
[novosirj@amarel-test1 novosirj]$ cat /etc/os-release 
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

### How did you install Singularity

Will attach script; doesn't leave anything to chance. Downloaded from source .tar.gz file.

[build-singularity-3.6.0-rc.3.sh.txt](https://github.com/hpcng/singularity/files/4637711/build-singularity-3.6.0-rc.3.sh.txt)
[build-singularity-3.6.0-rc.3-20200515.log](https://github.com/hpcng/singularity/files/4637708/build-singularity-3.6.0-rc.3-20200515.log)

Apologies in advance if I'm leaving anything out. I was recommended to run `git clean -fdx`, but I didn't install this via git. I _did_ use git to test a PR against -rc.1, do I guess it's possible something is left laying around somewhere, but I successfully built -rc.2 after that with no issues.

**Comment by LaHaine** [comment]
Please test with rc.4, this looks like an issue that was fixed there. (#5305 )

**Comment by novosirj** [comment]
Will do!

**Comment by dtrudg** [comment]
Yes... this should be fixed now...

```
=== RUN   TestSingularityActions
--- FAIL: TestSingularityActions (0.95s)
    actions_test.go:524: INFO:    Starting build...
        INFO:    Running post scriptlet
        [91mERROR  : Requesting capability set 0x0000003fffffffff while permitted capability set is 0x0000001fffffffff
        [0mFATAL:   While performing build: while running engine: exit status 1
        
    actions_test.go:525: unexpected failure: exit status 255
FAIL
coverage: 0.0% of statements
FAIL	github.com/sylabs/singularity/cmd/singularity	1.068s
```

That's the error that #5305 addresses.

**Comment by novosirj** [comment]
Confirmed to be fixed in 3.6.0-rc.4; works fine in our environment.
